components/menu/icons.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Icons
+page_title: Menu for Blazor | Icon
+description: Icons and images in the Menu for Blazor
+slug: menu-icons
+tags: telerik,blazor,menu,icon,iconclass,image
+published: True
+position: 2
```

Comment:
Move later in the tree on the left, it is not a very important feature, it should be around templates

Reply:
Done with fb6a044a0f7b443962247314e76ac0a28fe459fc